src/eduid_userdb/tests/test_user.py

```diff
@@ -362,3 +364,68 @@ def test_user_tou(self):
         user = User(data)
         self.assertTrue(user.tou.has_accepted('1'))
         self.assertFalse(user.tou.has_accepted('2'))
+
+    def test_locked_identity_load(self):
+        locked_identity = {
+            'created_by': 'test',
+            'created_ts': True,
+            'identity_type': 'nin',
+            'number': '197801012345'
+        }
+        data = self.data1
+        data['locked_identity'] = locked_identity
+        user = User(data)
+        self.assertTrue(user.locked_identity)
+        self.assertIsInstance(user.locked_identity.created_by, string_types)
+        self.assertIsInstance(user.locked_identity.created_ts, datetime.datetime)
+        self.assertIsInstance(user.locked_identity.identity_type, string_types)
+        self.assertIsInstance(user.locked_identity.number, string_types)
+
+        locked_identity_obj = LockedNinIdentityElement(locked_identity['number'], locked_identity['created_by'],
+                                                       locked_identity['created_ts'])
+        self.assertRaises(UserDBValueError, setattr, user, 'locked_identity', locked_identity_obj)
+
+    def test_locked_identity_set(self):
+        locked_identity = {
+            'created_by': 'test',
+            'created_ts': True,
+            'identity_type': 'nin',
+            'number': '197801012345'
+        }
+        user = User(self.data1)
+        locked_identity_obj = LockedNinIdentityElement(locked_identity['number'], locked_identity['created_by'],
+                                                       locked_identity['created_ts'])
+        user.locked_identity = locked_identity_obj
+        self.assertTrue(user.locked_identity)
+        self.assertIsInstance(user.locked_identity.created_by, string_types)
+        self.assertIsInstance(user.locked_identity.created_ts, datetime.datetime)
+        self.assertIsInstance(user.locked_identity.identity_type, string_types)
+        self.assertIsInstance(user.locked_identity.number, string_types)
+
+        self.assertRaises(UserDBValueError,  setattr, user, 'locked_identity', locked_identity_obj)
```

Comment:
easier to read like

with self.assertRaises(UserDBValueError):
    user.locked_identity = locked_identity_obj

also, I would call locked_identity_obj locked_nin.